meteor-app/imports/startup/client/routes.js

```diff
@@ -3,14 +3,15 @@ import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
 import isIframe from '/imports/helpers/isIframe'
 import Attendance from '/imports/ui/layouts/attendance'
 import MemberAddIframe from '/imports/ui/layouts/member-add-iframe'
+import Ordering from '/imports/ui/layouts/ordering'
 
 export const renderRoutes = () => (
   <Router>
     <Switch>
-      {
-        isIframe() &&
-        <Route component={MemberAddIframe} />
-      }
+    {// there is an iframe on the B2B wordpress site allowing registration to attendance app.
```

Comment:
React doesn't really like comments in the markup, although syntactically this is ok. Please move the commentary to a place just before the function `renderRoutes()`